Documentation/devel/release.md

```diff
@@ -0,0 +1,60 @@
+# rkt release guide
+
+How to perform a release of rkt.
+This guide is probably unnecessarily verbose, so improvements welcomed.
```

Comment:
@krnowak used here as a verb, not an adjective (cf "improvements [are]
encouraged")
Am 27.04.2015 05:56 schrieb "Krzesimir Nowak" <notifications@github.com>:

> In Documentation/devel/release.md
> <https://github.com/coreos/rkt/pull/813#discussion_r29143210>:
>
> > @@ -0,0 +1,60 @@
> > +# rkt release guide
> > +
> > +How to perform a release of rkt.
> > +This guide is probably unnecessarily verbose, so improvements welcomed.
>
> s/welcomed/welcome/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/813/files#r29143210>.
>